build.gradle

```diff
@@ -336,6 +333,9 @@ generateDocs.onlyIf { skipDocs() }
 task cleanGenerateDocs(type: Delete) {
     delete "${distTempDir}/temp"
     delete "${distTempDir}/docsrc/.gradle"
+    delete "${distTempDir}/docsrc/build"
+    delete "${distTempDir}/docsrc/node_modules"
+    delete "${distTempDir}/docsrc/out"
```

Comment:
@jelovirt Do we need these at all? 🤔 

These shouldn't exist in the `docsrc` submodule when the build runs, since they're generated at build time, and not versioned in the docs repo.

You added the `node_modules` exclusion in 5ad7508 so I'm guessing there was a reason, but I'm not sure it's necessary.

But I suppose it also does no harm to explicitly ensure these are never included.

Thoughts?

Reply:
They may exist when dist is run from a Git clone that has been used for development, i.e. developer workstation.